fdbclient/NativeAPI.actor.cpp

```diff
@@ -2784,6 +2788,10 @@ Future<Void> Transaction::commitMutations() {
 		if(options.firstInBatch) {
 			tr.flags = tr.flags | CommitTransactionRequest::FLAG_FIRST_IN_BATCH;
 		}
+		if(options.reportConflictingKeys) {
+			// TODO : Is it better to keep it as a flag?
+			tr.reportConflictingKeys();
```

Comment:
I think we have other cases where we access the flag directly out of options rather than copy it to the transaction, so that would probably be my preference.

Reply:
Not sure if I understand correctly.
I keep it in `CommitTransactionRef` since `CommitTransactionRef` will be directly copied to `ResolveTransactionBatchRequest` later.
If I use `flag` here, it is in `CommitTransactionRequest`, which needs a further step to be copied to `ResolveTransactionBatchRequest`.